docs/install/network_options.md

```diff
@@ -30,34 +30,13 @@ cp rke2-canal-config.yml /var/lib/rancher/rke2/server/manifests/
 
 For more information about the full options of the Canal config please refer to the [rke2-charts](https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-canal/charts/values.yaml).
 
-# Using Cilium instead of Canal
+# Using Cilium or Calico instead of Canal
 
-To use Cilium, Canal needs to be disabled on deployment. Refer to [Advanced Options and Configuration](../advanced.md#Disabling Server Charts) for information on how to do this.
+Starting with RKE2 v1.21, we can deploy Cilium or Calico as CNI plugin instead of Canal. To do so, pass `cilium` or `calico` as the value to the `--cni` flag. To override the default options, use a HelmChartConfig resource, as explained in the previous section for Canal options.
 
-To deploy Cilium, you can use the following config:
+For more information about the full options of the Cilium config, please refer to the [rke2-charts](https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-cilium/charts/values.yaml).
 
-```
-apiVersion: helm.cattle.io/v1
-kind: HelmChart
-metadata:
-  name: rke2-cilium
-  namespace: kube-system
-spec:
-  repo: https://rke2-charts.rancher.io
-  chart: rke2-cilium
-  bootstrap: true
-  valuesContent: |-
-    cilium: {}
-```
-
-The config needs to be copied over to the manifests directory:
-
-```
-mkdir -p /var/lib/rancher/rke2/server/manifests/
-cp rke2-cilium.yml /var/lib/rancher/rke2/server/manifests/
-```
-
-For more information about the full options of the Cilium config please refer to the [rke2-charts](https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-cilium/charts/values.yaml).
+For more information about the full options of the Calico config, please refer to the [calico installation resource](https://docs.projectcalico.org/reference/installation/api#operator.tigera.io/v1.Installation)
```

Comment:
I think we should also point users to the rke2-charts values.yaml for Calico, since their documentation seems to expect that the users is configuring the system via the Installation CRD, and it is unclear even to me how to translate from helm chart values to Installation CRD fields.

Reply:
Yeah I guess I'd just link directly to the patched values file on the main branch for both cilium and calico:
https://github.com/rancher/rke2-charts/blob/main/charts/rke2-calico/rke2-calico/v3.18.1-103/values.yaml

Reply:
good idea! I did not know the main branch shows values.yaml